ctapipe/image/hillas.py

```diff
@@ -146,11 +127,11 @@ def hillas_parameters_1(pix_x, pix_y, image):
     a = d2 / d1
     # Angle between ellipse major ax. and x-axis of camera. Will be used for
     # disp
-    delta = np.pi / 2.0 + np.arctan(a)
+    delta = ((np.pi / 2.0) + np.arctan(a)) * u.rad
```

Comment:
The motivation to introduce units to delta is because it is returned in degrees as psi. But it is true that this just slows down the computation. To make more sense, I have just now made a new commit where the entire code uses the angle values in radians and at the return line deals with all the units and unit conversions. 

And also, I realized the name 'delta' might create confusion. Hence I renamed it to psi, since it is returned as psi in the output.